taskboot/github.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+from github import Github
+from github import UnknownObjectException
+
+from taskboot.config import Configuration
+from taskboot.utils import load_named_artifacts
+
+logger = logging.getLogger(__name__)
+
+
+def build_release_notes(repository, tag):
+    signature = (
+        "\n---\nReleased with [mozilla/task-boot](https://github.com/mozilla/task-boot)"
+    )
+
+    # Get all commits between both versions using the comparison endpoint
+    latest_release = repository.get_latest_release()
+    diff = repository.compare(latest_release.tag_name, tag.ref)
```

Comment:
Does this work when there is no previous release?

Reply:
:sob:

Reply:
Now it does ! https://github.com/La0/test-taskboot/releases/tag/0.1-rc1